### PR TITLE
Fix hot reloading

### PR DESCRIPTION
**Pull request by gomorizsolt**

### CODE REVIEW

File: sandbox.config.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-    "template": "static"
+    "template": "static",
+    "hardReloadOnChange": true
```
Comment (thisismydesign) [inline sandbox.config.json:3]: Cool! Wouldn't this be enough though? Or the live server?

Comment (gomorizsolt) [reply sandbox.config.json:3]: Eh, no, it doesn't solve the issue.

Comment (gomorizsolt) [reply sandbox.config.json:3]: Neither does `live-server`.

Comment (gomorizsolt) [reply sandbox.config.json:3]: I think we must use a different template (e.g. the one that uses `parcel` under the hood) although I still don't get why `live-server` doesn't refresh the embedded version.

Comment (gomorizsolt) [reply sandbox.config.json]: > I think we must use a different template (e.g. the one that uses `parcel` under the hood) although I still don't get why `live-server` doesn't refresh the embedded version.

Let's use `parcel` instead then: https://codesandbox.io/s/github/gomorizsolt/firebase-auth-starter/tree/fix-hot-reloading

Now this should pick up the changes right away as you edit the code (tested the embedded version as well).

Comment (gomorizsolt) [reply sandbox.config.json]: @thisismydesign https://github.com/componently-com/flying-forms-html-form-starter

This starter doesn't refresh on changes either, right?

Comment (gomorizsolt) [reply sandbox.config.json]: Please let me know if hot-reloading should also be enabled on these starters:
- https://github.com/componently-com/flying-forms-html-form-starter
- https://github.com/componently-com/algolia-starter
- https://github.com/componently-com/spotify-web-playback-starter

Comment (thisismydesign) [reply sandbox.config.json]: Thanks! This one is enough for now.

**Pull request merged**